input/include/NovelRT/Input/NovelKey.hpp

```diff
@@ -4,20 +4,14 @@
 #ifndef NOVELRT_INPUT_NOVELKEY_H
 #define NOVELRT_INPUT_NOVELKEY_H
 
-#ifndef NOVELRT_INPUT_H
-#error NovelRT does not support including types explicitly by default. Please include Input.h instead for the Input namespace subset.
-#endif
-
```

Comment:
If we are still relying on a centralised header as per your last comment, we should probably revert this

Reply:
Resolved - i have banished the centralised header to the shadow realm.